native/package.json

```diff
@@ -17,12 +17,12 @@
     ]
   },
   "devDependencies": {
-    "@storybook/react-native": "^3.2.11",
-    "babel-preset-react-native": "3.0.2",
-    "react-dom": "^16.0.0",
-    "react-test-renderer": "^16.0.0",
+    "babel-preset-react-native": "^4.0.0",
+    "@storybook/react-native": "^3.2.12",
     "react": "^16.0.0",
-    "react-native": "^0.47.0"
+    "react-dom": "^16.0.0",
```

Comment:
The peer for `0.49.3` is `16.0.0.beta-5` for `react`, `react-dom`, `react-test-renderer`. We should specify those versions IMO

Reply:
cool will do

Reply:
Fixed - here is an example https://runkit.com/embed/umidffgbnot2

Reply:
Please approve when happy :)

Reply:
Ah you mean in our root dependencies? I guess so ...